Bottom: 4019690013ee59e234a7b014331f152e47ecc42f
Top:    6219f58ae8376cd7afbb8acf46e5d540911e9a40
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-02 14:49:19 +0100

autoroute.c: Temporarily turn on some debugging drawing


---

diff --git a/src/autoroute.c b/src/autoroute.c
index 1178292..bcc34eb 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -94,10 +94,10 @@ RCSID ("$Id$");
 
 /*
 #define ROUTE_DEBUG
-//#define DEBUG_SHOW_ROUTE_BOXES
+#define DEBUG_SHOW_ROUTE_BOXES
 #define DEBUG_SHOW_EXPANSION_BOXES
-//#define DEBUG_SHOW_EDGES
-//#define DEBUG_SHOW_VIA_BOXES
+#define DEBUG_SHOW_EDGES
+#define DEBUG_SHOW_VIA_BOXES
 #define DEBUG_SHOW_TARGETS
 #define DEBUG_SHOW_SOURCES
 //#define DEBUG_SHOW_ZIGZAG
@@ -4191,7 +4191,7 @@ RouteOne (routedata_t * rd, routebox_t * from, routebox_t * to, int max_edges)
       /* we should never add edges on inactive layer groups to the heap. */
       assert (is_layer_group_active[e->rb->group]);
 #if defined(ROUTE_DEBUG) && defined(DEBUG_SHOW_EXPANSION_BOXES)
-      //showedge (e);
+      showedge (e);
 #endif
       if (e->rb->flags.is_thermal)
 	{
